tests/e2e/test_actions.py

```diff
@@ -94,3 +95,48 @@ def test_actions_sanity():
     assert (
         len(result.stdout.splitlines()) == 44
     ), "unexpected output from the rulebook"
+
+
+@pytest.mark.e2e
+def test_run_playbook():
+    """
+    Execute a rulebook that contains multiple run_playbook actions
+    to validate all params that are available with the action.
+    """
+
+    rulebook = utils.BASE_DATA_PATH / "rulebooks/actions/test_run_playbook.yml"
+    os.environ["SOURCE_SHUTDOWN_AFTER"] = "6"
```

Comment:
This set the environment variable for the main process, generating potential side effects. You must create a copy of the environment, update it and inject it only for this specific command. Example: https://github.com/ansible/event-driven-ansible/blob/main/tests/integration/event_source_webhook/test_webhook_source.py#L44

Reply:
@Alex-Izquierdo done. Pls re-review